charts/dapr/charts/dapr_scheduler/templates/dapr_scheduler_statefulset.yaml

```diff
@@ -0,0 +1,256 @@
+{{- if (eq .Values.global.scheduler.enabled true) }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: dapr-scheduler-server
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: dapr-scheduler-server
+    {{- range $key, $value := .Values.global.k8sLabels }}
+    {{ $key }}: {{ tpl $value $ }}
+    {{- end }}
+spec:
+{{- if eq .Values.scaleZero true }}
+  replicas: 0
+{{- else if or (eq .Values.global.ha.enabled true) (eq .Values.ha true) }}
```

Comment:
The distributed scheduler is different, we expect users to increase the number of replicas. So, instead of using "ha" to hard-code the quantity, we should take the number of replicas, defaulting to 1.

Reply:
I switched the ha checks to be: 
```
{{- if or (eq .Values.global.ha.enabled true) (gt (int .Values.replicaCount) 1) }}

```

respecting the global ha val, but also allowing a `replicaCount` for scheduler that defaults to 1. lmk if you want it changed differently

Reply:
resolved